.bazelrc

```diff
@@ -69,7 +69,11 @@ build:unix --per_file_copt='external/zlib[~/].*\.c@-std=c90' --host_per_file_cop
 build:unix --@capnp-cpp//src/kj:openssl=True --@capnp-cpp//src/kj:zlib=True --@capnp-cpp//src/kj:brotli=True --@capnp-cpp//src/kj:libdl=True
 
 build:linux --config=unix
-build:macos --config=unix
+
+# macOS needs additional build options for symbols to work correctly
```

Comment:
I have previously built workerd through the command line for the most part, where `spawn_strategy ` wasn't being set before. This will disable the sandbox across mac builds, I guess that's a tradeoff we're willing to accept?
PR LGTM otherwise

Reply:
Good point, let's move that back so it's just for vscode debugging and have this called out in the comment there.

Reply:
Fixed in dd3481c369f95b5f18812c2895a87377abc0799b.